gno.land/pkg/sdk/vm/handler.go

```diff
@@ -107,34 +102,9 @@ func (vh vmHandler) Query(ctx sdk.Context, req abci.RequestQuery) abci.ResponseQ
 				secondPart(req.Path), req.Path)))
 	}
 
-	// Log the telemetry
-	logQueryTelemetry(path, res.IsErr())
-
 	return res
 }
 
-// logQueryTelemetry logs the relevant VM query telemetry
-func logQueryTelemetry(path string, isErr bool) {
```

Comment:
why remove this?

Reply:
We removed that with @zivkovicmilos because so far we find out that collecting VM low-level info are not relevant in terms of telemetry.

Reply:
It's an absolutely useless metric that's triggered at non-standard times (by users). Tracing VM query handles is a better alternative than a metric